### DISCUSSION TITLE
When installing automatically add the package to path variable

### DISCUSSION THREAD

**Discussion by AutoAFK** [microsoft/winget-cli#5091] [Q&A] [answered]
Is it even possible?
I can do it manually but after coming from `scoop` it just seems like an annoying task.
Until now I only found out that you can start interactive download but I want to know
if there is any other way?

**Comment by denelon** [comment, marked as answer]
It can be done if winget is called via cmd.exe as opposed to PowerShell.

The remaining work for this to function in PowerShell is tracked with:
* https://github.com/microsoft/winget-cli/issues/549
